src/Orchard.Web/Modules/Orchard.Taxonomies/Views/EditorTemplates/Fields/TaxonomyFieldList.cshtml

```diff
@@ -5,16 +5,19 @@
     Script.Require("jQuery");
     var Taxonomyprefix = ((ViewData)).TemplateInfo.HtmlFieldPrefix.Replace('.', '_');
     using (Script.Foot()) {
-        <script type="text/javascript">
+    <script type="text/javascript">
             //<![CDATA[
             $(document).ready(function () {
                 var pageurl = '@Url.Action("GetTaxonomy", "LocalizedTaxonomy", new { area = "Orchard.Taxonomies" })';
-                function filterTaxonomyCulture(culture) {
+                function @("filterTaxonomyCulture_" + Model.PartName + "_" + Model.FieldName)(culture) {
```

Comment:
this function name could be created once in a `var` in this view

Reply:
Suggestion applied, thanks.